lint/highlight.py

```diff
@@ -400,6 +433,28 @@ def draw(self, view):
                 region = sublime.Region(self.newlines[line], self.newlines[line])
                 gutter_regions[error_type].append(region)
 
+        def hide_phantoms(href):
```

Comment:
I meant to do this:
```
@staticmethod
def hide_phantoms(view):
  view.erase_phantoms('sublimelinter')
```
so that you have one method to remove phantoms. That way, if some handling of phantoms is changed, you only have to change one place.